foreman-installer/README

```diff
@@ -0,0 +1,2 @@
+how to build this package:
```

Comment:
I'm not that familiar with the build process and to me this line implies a scratch build while the next implies a release (including a tag). What does this exactly achieve?